engines/sci/engine/script_patches.cpp

```diff
@@ -5596,30 +5596,30 @@ static const SciScriptPatcherEntry laurabow2Signatures[] = {
 static const uint16 mothergoose256SignatureReplay[] = {
 	0x7a,                            // push2
 	0x78,                            // push1
-	0x5b, 0x00, 0xbe,                // lea global[BEh]
+	0x5b, 0x00, 0xbe,                // lea global[190]
 	0x36,                            // push
 	0x43, 0x70, 0x04,                // callk MemorySegment
 	0x7a,                            // push2
-	0x5b, 0x00, 0xbe,                // lea global[BEh]
+	0x5b, 0x00, 0xbe,                // lea global[190]
 	0x36,                            // push
 	0x76,                            // push0
 	0x43, 0x62, 0x04,                // callk StrAt
-	0xa1, 0xaa,                      // sag global[AAh]
+	0xa1, 0xaa,                      // sag global[170]
 	0x7a,                            // push2
-	0x5b, 0x00, 0xbe,                // lea global[BEh]
+	0x5b, 0x00, 0xbe,                // lea global[190]
 	0x36,                            // push
 	0x78,                            // push1
 	0x43, 0x62, 0x04,                // callk StrAt
 	0x36,                            // push
 	0x35, 0x20,                      // ldi 20
 	0x04,                            // sub
-	0xa1, SIG_ADDTOOFFSET(+1),       // sag global[57h] -> FM-Towns [9Dh]
+	0xa1, SIG_ADDTOOFFSET(+1),       // sag global[87] -> 57h, FM-Towns = 9Dh
 	// 35 bytes
 	0x39, 0x03,                      // pushi 03
-	0x89, SIG_ADDTOOFFSET(+1),       // lsg global[1Dh] -> FM-Towns [1Eh]
+	0x89, SIG_ADDTOOFFSET(+1),       // lsg global[29] -> 1Dh, FM-Towns = 1Eh
```

Comment:
This right here gets now really confusing, while in retrospect it was already quite confusing already and should have been worded differently.

like for example:
// sag global[57h], for FM-Towns: sag global[90h]